curriculum/js1/10.js

```diff
@@ -7,7 +7,9 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
+const solution = (a, b, C) => {
+  setTimeout (C, a*1000)
+  setTimeout (C, (a+b)*1000)
```

Comment:
Lines 11 & 12:
* Please remove this space after `setTimeout`. `setTimeout` is a method/function. Common style guides will specify no space between the a function name and the opening bracket.

* Please add semicolons. JavaScript will let you get away with this, but most people will expect to see semicolons after these function calls.

Reply:
Hi Daniel,

Thanks for the comments. I made changes and pushed the branch again to github just now.

Can you review again? Also, do I have to make a new merge request?

Thanks!